mono/metadata/w32socket-unix.c

```diff
@@ -130,25 +126,6 @@ mono_w32socket_initialize (void)
 	mono_w32handle_register_ops (MONO_W32HANDLE_SOCKET, &ops);
 }
 
-static gboolean
-cleanup_close (gpointer handle, gpointer data, gpointer user_data)
```

Comment:
I understand the reasoning for removing handles cleanup, I don't know know about this one.

Besides that. The commit messages for both removals are crap, they *must* state the reasoning behind the removal.